client/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 12.2
```

Comment:
Actually, these features were introduced in [12.3](https://github.com/guardian/content-api-scala-client/blob/v12.3/client/CHANGELOG.md).